modules/nw-networkpolicy-create.adoc

```diff
@@ -1,59 +1,120 @@
 // Module included in the following assemblies:
 //
 // * networking/network_policy/creating-network-policy.adoc
-// * networking/configuring-networkpolicy.adoc
 // * post_installation_configuration/network-configuration.adoc
 
+ifeval::["{context}" == "configuring-multi-network-policy"]
+:multi:
+endif::[]
+
 [id="nw-networkpolicy-create_{context}"]
 
 = Creating a network policy
 
-To define granular rules describing ingress network traffic allowed for projects
-in your cluster, you can create a network policy.
+To define granular rules describing ingress network traffic allowed for namespaces in your cluster, you can create a network policy.
+
+[NOTE]
+====
+If you log in with a user with the `cluster-admin` role then you can create a network policy in any namespace in the cluster.
+====
 
 .Prerequisites
 
-* Your cluster is using a default CNI network provider that supports `NetworkPolicy` objects, such as the OpenShift SDN network provider with `mode: NetworkPolicy` set. This mode is the default for OpenShift SDN.
 * You installed the OpenShift CLI (`oc`).
-* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You are logged in to the cluster with a user with `admin` privileges.
+* You are working in the namespace that the network policy applies to.
+ifndef::multi[]
+* Your cluster is using a cluster network provider that supports `NetworkPolicy` objects, such as the OpenShift SDN network provider with `mode: NetworkPolicy` set. This mode is the default for OpenShift SDN.
+endif::multi[]
 
 .Procedure
 
 . Create a policy rule:
-.. Create a `<policy-name>.yaml` file where `<policy-name>` describes the policy
-rule.
-.. In the file you just created define a policy object, such as in the following
-example:
+.. Create a `<policy_name>.yaml` file:
++
+[source,terminal]
+----
+$ touch <policy_name>.yaml
+----
 +
+--
+where:
+
+`<policy_name>`:: Specifies the policy rule file name.
+--
+
+.. Define a policy object in the file that you just created, such as in the following examples:
++
+.Deny ingress from all pods in all namespaces
 [source,yaml]
 ----
+ifndef::multi[]
 kind: NetworkPolicy
 apiVersion: networking.k8s.io/v1
```

Comment:
Should it be apiVersion: k8s.cni.cncf.io/v1beta1?

Reply:
@weliang1, yes, I've added a conditional for it because this file is included in 2 different assemblies.